collectors/deployment.go

```diff
@@ -160,4 +180,7 @@ func (dc *deploymentCollector) collectDeployment(ch chan<- prometheus.Metric, d
 	} else {
 		addGauge(descDeploymentStrategyRollingUpdateMaxUnavailable, float64(maxUnavailable))
 	}
+
+	labelKeys, labelValues := kubeLabelsToPrometheusLabels(d.Labels)
+	addGauge(deploymentLabelsDesc(labelKeys), 1, labelValues...)
```

Comment:
Just move these two lines to be the first metrics to be added (line 164). The reason these metrics are not showing up is the guard clause on line 173-175.